tests/Tests.FeatureManagement/FeatureManagement.cs

```diff
@@ -14,6 +14,7 @@
 using System.Linq;
 using System.Net;
 using System.Net.Http;
+using System.Reflection;
```

Comment:
Sorry, I may miss it as it's not very easy to tell. What we are using in the reflection namespace?